rotkehlchen/chain/arbitrum_one/modules/rainbow/decoder.py

```diff
@@ -0,0 +1,25 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.evm.decoding.rainbow.constants import RAINBOW_ROUTER_CONTRACT
+from rotkehlchen.chain.evm.decoding.rainbow.decoder import RainbowCommonDecoder
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.chain.evm.node_inquirer import EvmNodeInquirer
+    from rotkehlchen.user_messages import MessagesAggregator
+
+
+class RainbowDecoder(RainbowCommonDecoder):
+
+    def __init__(
+            self,
+            evm_inquirer: 'EvmNodeInquirer',
+            base_tools: 'BaseDecoderTools',
+            msg_aggregator: 'MessagesAggregator',
+    ) -> None:
+        super().__init__(
+            evm_inquirer=evm_inquirer,
+            base_tools=base_tools,
+            msg_aggregator=msg_aggregator,
+            router_address=RAINBOW_ROUTER_CONTRACT,
```

Comment:
since this never changes, why not define it in `RainbowCommonDecoder`?

Reply:
For Zora the address if different
https://github.com/rainbow-me/swaps/tree/main/smart-contracts#deployment-addresses

Reply:
at the moment, we don't support zora.